bin/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+REVISION=$(git show-ref origin/master | cut -f 1 -d ' ')
+TAGGED_IMAGE=gcr.io/${GOOGLE_PROJECT}/dispatch:${REVISION}
+gcloud --quiet container images describe ${TAGGED_IMAGE} || { status=$?; echo "Container not finished building" >&2; exit $status; }
+
+gcloud --quiet container images add-tag ${TAGGED_IMAGE} gcr.io/${GOOGLE_PROJECT}/dispatch:latest
+
+kubectl set image deployment/libraries-dispatch libraries-dispatch-container=${TAGGED_IMAGE}
```

Comment:
```suggestion
TAGGED_IMAGE=librariesio/dispatch:${REVISION}

kubectl set image deployment/dispatch-service dispatch-container=${TAGGED_IMAGE}
```

Since we're on docker hub, we can't use gcloud stuff and updating for what the deployment and container are called